packages/apps/tools/src/components/Global/AccountNameField/index.tsx

```diff
@@ -43,44 +31,13 @@ export const AccountNameField: FC<IAccountNameFieldProps> = ({
   ...rest
 }) => {
   const elementId = 'kd-select-account-input';
-  const [mode, setMode] = useState<'input' | 'select'>('input');
+  const mode = 'input';
   const { t } = useTranslation('common');
   const { onChange, ...restInputProps } = inputProps;
 
-  const {
-    accounts,
-    selectedAccount,
-    selectedNetwork,
-    setSelectedAccount,
-    session,
-  } = useWalletConnectClient();
-
-  useDidUpdateEffect(() => {
-    if (session && accounts?.length) setMode('select');
-    else setMode('input');
-  }, [session, accounts]);
+  const { selectedAccount, setSelectedAccount } = useWalletConnectClient();
 
   const lookup = {
-    select: (
```

Comment:
Should we really delete this? 🤔

Reply:
Didn't want to leave it commented out for later, but if you think that's better I can bring this back (for now we don't have the accounts from the wallet)

Reply:
I mean I guess we can always get them back by referencing this PR, so it's fine like this 👍